src/styles/blocks/header.scss

```diff
@@ -0,0 +1,16 @@
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px #eee;
```

Comment:
Hex color values should use lowercase letters for consistency as per the code style rules. Change `#eee` to `#EEE`.